weave/weave_client.py

```diff
@@ -664,12 +736,6 @@ def ref_value_input_to(self, ref: "ref_base.Ref") -> list[Call]:
     def ref_output_of(self, ref: ObjectRef) -> typing.Optional[Call]:
         raise NotImplementedError()
 
-    def add_feedback(self, run_id: str, feedback: typing.Any) -> None:
```

Comment:
was `add_feedback` implemented elsewhere? Why remove if not?

Reply:
nvm, i see it is implemented on `RefFeedback`

Reply:
Yeah, this isn't necessary for the new APIs, just me trying to cleanup some unimplemented stuff that seemed like it could cause confusion.